src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Symbolic/Algebras/BV64Algebra.cs

```diff
@@ -211,7 +188,17 @@ public string PrettyPrint(ulong bv)
             {
                 var sb = new StringBuilder();
                 sb.Append('[');
```

Comment:
The following should work instead for `PrettyPrint`, because now `CharSetSolver` is never `null` and `_partition` is never `null` in the algebra, because the algebra cannot arise via deserialization any more, and this will print the true meaning of the label (not the abstract one based on `ulong` representation of the minterms):
`public string PrettyPrint(ulong bv) => CharSetSolver.PrettyPrint(ConvertToCharSet(CharSetSolver, bv));`

Reply:
Thanks, changed.